.github/workflows/schemalint.yaml

```
@@ -0,0 +1,22 @@
+name: Schemalint
```

As the tool's name is `schemalint`, I would prefer the action name to use the same spelling.

```suggestion
name: schemalint
```